sessions/overview-of-available-tools.qmd

```diff
@@ -7,12 +7,37 @@ order: 12
 
 The aim of this session is to try out some of the tools that are available that use some of the ideas we have discussed in this course.
 
-# Suggested tools
+# Tools vs custom approaches
+
+In the course so far we introduced key concepts and methods in nowcasting and forecasting of infectious disease dynamics, using a combination of R and (mostly) stan.
+Being able to understand these concepts and implement them in stan comes with the ability to create and adapt models that are tailor made to any given situation and specific characteristics of any given data set.
+At the same time, there is value in using and contributing to open-source **tools** that implement some or all of the methods we have encountered in the course.
+These tools have varying levels of flexibility to deal with different kinds of epidemiological problems and data.
+Whilst they will never have the flexibility of completely custom approaches, using and contributing to them avoids duplication, improves the chances of finding errors, and helps discussing and ultimately enforcing best practice.
+
+We are going to suggest trying three tools that implement several or all the concepts introduced in this course:
+
+## EpiEstim
+
+[EpiEstim](https://mrc-ide.github.io/EpiEstim/) implements the renewal equation on a time series of infections in a Bayesian framework, i.e. the model in `estimate-r.stan` in the [session on R estimation](R-estimation-and-the-renewal-equation#estimating-R-from-a-time-series-of-infections).
+In combination with the [projections](https://www.repidemicsconsortium.org/projections/) package in can be used for forecasting.
+
+The [EpiEstim vignette](https://mrc-ide.github.io/EpiEstim/articles/full_EpiEstim_vignette.html) is a good starting point for a walkthrough of $R_t$ estimation and forecasting.
 
 ## EpiNow2
 
+[EpiNow2](https://epiforecasts.io/EpiNow2/dev/) implements the renewal equation on a time series of delayed outcomes including nowcasts with a known reporting delay distribution, as well as forecasts using a Gaussian Process or random walk model, i.e. the models introduced here except the joint inference of nowcasts and delays and/or reproduction numbers.
+
+The [example of nowcasting, Rt estimation and forecasting with EpiNow2](https://github.com/epiforecasts/nowcasting.example/blob/main/inst/reports/epinow2.md) is a good place to find out more about how to use EpiNow2 with linelist data.
+
 ## Epinowcast
 
-## EpiEstim
+[Epinowcast](https://package.epinowcast.org/) was created to replace EpiNow2.
+It implements all the models discussed in the course except for the ability to make forecasts.
```

Comment:
It can do this but only via inflating the input data and likelihood skipping
```suggestion
It implements all the models discussed in the course. _Note that whilst it can forecast this does not currently have a user-facing interface._
```